core/model/modx/processors/element/tv/duplicate.class.php

```diff
@@ -104,5 +105,43 @@ public function duplicateMediaSources() {
             }
         }
     }
+
+    /**
+     * Get the new caption for the duplicate
+     * @return string
+     */
+    public function getNewCaption()
+    {
+        $caption = $this->getProperty($this->captionField);
+        $newCaption = !empty($caption)
+            ? $caption
+            : $this->modx->lexicon('duplicate_of', array('name' => $this->object->get($this->captionField)));
```

Comment:
When there is no caption (in example because we removed it with the UI when duplicating and the source has no caption), this forces a caption of `Duplicate of `.